test/disabled/id-tests.js

```diff
@@ -1,76 +0,0 @@
-describe( 'Dynamic ids work properly', function(){
```

Comment:
looks like the concept of 'dynamic ids' has been removed, so simply deleting these tests